sdk/src/feature_set.rs

```diff
@@ -43,7 +43,7 @@ pub mod sha256_syscall_enabled {
 }
 
 pub mod no_overflow_rent_distribution {
-    solana_sdk::declare_id!("4kpdyrcj5jS47CZb2oJGfVxjYbsMm2Kx97gFyZrxxwXz");
+    solana_sdk::declare_id!("9UQYJwv7RXK3mQQzAaxXcYmdVfii3xAGFMkuYnzJUwMX");
```

Comment:
@ryoqun  I repurposed this existing feature, hopefully that is ok!

Reply:
If I'm correct, I think you don't need feature gating for this, meaning less work and no feature activation ops. https://github.com/solana-labs/solana/pull/13569/files#r526635538

Also, I have concerns for repurposing this:
- this isn't still enabled on mb (but will be enabled soon).
- this will made replaying past slots by `ledger-tool` a bit harder. I think we should at least wait a month or so in that regard.